addons/mail/static/src/discuss/core/common/avatar_stack.js

```diff
@@ -36,7 +38,9 @@ export class AvatarStack extends Component {
         }
         // Compute cumulative offset,
         const marginDirection = this.props.direction === "v" ? "top" : "left";
-        style += `margin-${marginDirection}: -${this.props.size / 3}px`;
+        if (this.props.overlap) {
+            style += `margin-${marginDirection}: -${this.props.size / 3}px`;
```

Comment:
:pick: I feel like we should have a small margin when overlap is `false`. Otherwise items are almost touching.

Reply:
What about this? This is same margin but positive rather than negative when no overlap:
<img width="56" alt="Screenshot 2025-02-20 at 11 07 19" src="https://github.com/user-attachments/assets/6b001a14-0c56-4fc5-9ce4-e893be938c4e" />